BeyondTrust/beyondtrust-pra/ingest/parser.yml

```diff
@@ -69,6 +78,7 @@ stages:
           user.target.id: "{{parsed_event.message.data.user_id}}"
           user.target.name: "{{parsed_event.message.data.username}}"
           process.executable: "{{parsed_event.message.data.exeName}}"
+          process.command_line: "{{parsed_event.message.data.text}}"
```

Comment:
I propose setting this field only if `event_type == "Remote Shell Event"`.
I guess that `parsed_event.message.data.text` can be used on other purpose for some other types